content/applications/sales/point_of_sale/shop/serial_numbers.rst

```diff
@@ -0,0 +1,64 @@
+==============================
+Serial numbers and lots in PoS
+==============================
+
+Working with **serial numbers** and **lots** allows tracking your products' movements. When products
+are tracked, the system identifies their location based on their last movement.
+
+To enable traceability, go to :menuselection:`Point of Sale --> Products --> Products`. Then,
+select a product and check the :guilabel:`Tracking By Unique Serial Number` or the
+:guilabel:`Tracking By Lots` box in the :guilabel:`Inventory` tab.
+
+.. image:: serial_numbers/product-form-traceability.png
+   :align: center
+   :alt: Enable traceability settings
+
+Serial numbers and lots import
+==============================
+
+You can import serial numbers in Point of Sale. To do so, select a **sales order** or a
+**quotation** containing tracked products. Then, agree to load the **Lots or Serial Numbers** linked
+to the :abbr:`SO (sales order)`.
+
+.. image:: serial_numbers/importing-sn.png
+   :align: center
+   :width: 480
+   :alt: Pop-up window for serial number import
+
+.. seealso::
+   - :doc:`../shop/sales_order`
```

Comment:
I'd rather put the seealso section at the end of the section, just before the next heading. Here it kind of breaks the reading.